packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 50.1.0 - October 14, 2020
```

Comment:
@rcoringrato-sfdc these are the items I think should be part of this week's changelog:

https://github.com/forcedotcom/salesforcedx-vscode/pull/2619
https://github.com/forcedotcom/salesforcedx-vscode/pull/2607
https://github.com/forcedotcom/salesforcedx-vscode/pull/2591
https://github.com/forcedotcom/salesforcedx-vscode/pull/2573